.github/workflows/moodle-release.yml

```diff
@@ -2,18 +2,18 @@
 # Whenever version.php is changed, add the latest version
 # to the Moodle Plugins directory at https://moodle.org/plugins
 #
-name: MOODLE_310_STABLE - Releasing in the Plugins directory
```

Comment:
Might be worth combining the release logic into ci.yml (per the latest reusable workflows docs)